### PR TITLE
Add Guillotina GH action

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,11 +34,6 @@ jobs:
       # node install
       - run: yarn install --frozen-lockfile
 
-      # node code analysis
```
Comment (tisto) [inline .github/workflows/ci.yml:37]: @sneridagh why did you remove the code analysis section here?

Comment (sneridagh) [reply .github/workflows/ci.yml:37]: yes, because it has its own GH action, right?

Comment (tisto) [reply .github/workflows/ci.yml:37]: Not sure if this is the way to go. gh actions can only run jobs in parallel. therefore we'd rather remove the code-analysis job again instead of the code-analysis section from the main job.

Comment (sneridagh) [reply .github/workflows/ci.yml:37]: I thought that was the intended thing to go, whatever you think it's the best way to go.

Travis is really starting to annoy me. I guess they are starting to let it go.

**Comment by sneridagh** [comment]
@tisto Should we remove the static separate job then?

**Comment by tisto** [comment]
@sneridagh yeah. I guess so. Maybe at some point gh actions provide sequential pipeline steps.
